base-2018/partials/_sub_field_blocks.twig

```diff
@@ -64,14 +64,14 @@
 
     {# Special case for 'select' fields: if it's a multiple select, the value is an array. #}
     {% if fieldtype == "select" and value is not empty %}
-        <p><strong>{{ key }}: </strong>
+        <p><strong>{% if record.contenttype.fields[key].label is empty %}{{ key }}{% else %}{{ record.contenttype.fields[key].label }}{% endif %}: </strong>
```

Comment:
I think this can be simplified to `record.contenttype.fields[key].label|default(key)`

Reply:
Yes, this works. You found a better and shorter solution. Thanks.